content/specs/versions-macos-xcode-13-3.md

```diff
@@ -0,0 +1,134 @@
+---
+description: A list of tools available out-of-the-box on Codemagic build machines.
+title: macOS build machine specification (Xcode 13.3+)
+aliases:
+  - '../releases-and-versions/versions5'
+  - '../specs/versions5'
+weight: 9
+---
+
+## Hardware
+
+- Standard VM on Mac mini `2.3GHz Quad Core / 8GB`
+- Premium VM on Mac Pro `3.7GHz Quad Core / 32GB`
+
+## System
+
+- System version `macOS 12.2 (21D49)`
+- Kernel version `Darwin 21.3.0`
+- Disk `322GB (Free Space: 149GB)`
+
+## Pre-installed tools
+
+- Android tools `/usr/local/share/android-sdk`
+- Android NDK `23.1.7779620`
+- aws `2.4.15`
+- cocoapods `1.11.2`
+- cordova `11.0.0`
+- curl
+- docker `20.10.2`
+- fastlane `2.203.0`
+- firebase `9.3.0`
+- flutter `2.8.1 ($HOME/programs/flutter)`
+- gem `3.3.6`
+- gh `2.4.0`
+- git `2.35.1`
+- Google Cloud SDK `327.0.0`
+- gradle `7.3.1`
+- gsutil `4.58`
+- homebrew `3.3.13`
+- ionic `6.12.4`
+- jq
+- ktlint `0.43.2`
+- node `14.15.5`
+- npm `8.3.1`
+- python `3.8.7`
+- python3 `3.8.7`
+- ruby `2.7.2p137`
+- ssh
+- sudo
+- tar
+- unzip
+- wget
+- yarn `1.22.17`
+- yq
+- zip
+
+## Android emulators
+
+- **emulator**
+
+    - Device: `pixel_3a (Google)`
+    - Path: `/Users/builder/.android/avd/emulator.avd`
+    - Target: `Google Play (Google Inc.)`
+    - Based on: `Android 10.0 (Q)`
+    - Tag/API: `google_apis_playstore/x86`
+    - Skin: `pixel_3a`
+    - Sdcard: `512M`
+
+## Java versions
+
+- **17.0.2** JAVA_HOME: `/Library/Java/JavaVirtualMachines/zulu-17.jdk/Contents/Home`
+- **16.0.2** JAVA_HOME: `/Library/Java/JavaVirtualMachines/zulu-16.jdk/Contents/Home`
+- **11.0.14** (default) JAVA_HOME: `/Library/Java/JavaVirtualMachines/zulu-11.jdk/Contents/Home`
+- **1.8.0_322** JAVA_HOME: `/Library/Java/JavaVirtualMachines/zulu-8.jdk/Contents/Home`
+
+## Xcode 13.3 (13E5086k)
+
+This is the Xcode version used by default when you select `13.3` in build settings in the workflow 
+editor for Flutter apps or set Xcode version to `13.3` in your codemagic.yaml file. 

```

Comment:
13.3 will become the edge version instead of the 13.2 (needs updating on both pages)

Reply:
Addressed in 26fefbe.